docs/features/readonly-variables-and-parameters.md

```diff
@@ -0,0 +1,61 @@
+### High Level Language Proposal:  
+Allow 'readonly' to be used on local variables and on parameters. 
+
```

Comment:
Why readonly and not const? We already allow const locals as long as they are of primitive types. Readonly locals seem to be semantically exactly the same, just supporting wider range of types. 
Perhaps it is better to generalize existing concept, than introduce a new one which overlaps the former?